### ISSUE TITLE
It is not possible to activate AA Tacan of tanker using MOOSE

### ISSUE THREAD

**Issue by gronank** [FlightControl-Master/MOOSE#1732]
Inside BEACON.lua:257 __function BEACON:AATACAN( TACANChannel, Message, Bearing, BeaconDuration )__
there's a call to SetCommand with the argument
```
{
      id = "ActivateBeacon",
      params = {
        type = 4,
        system = System,
        callsign = Message,
        frequency = Frequency,
      },
}
```
This does not seem to work. If you assign tacan to a tanker in the mission editor and look at what is written to the mission file you'll see the following:
```
{ 
  id = 'ActivateBeacon', 
  params = { 
    ["type"] = 4,
    --["AA"] = false,
    ["callsign"] = "TKR",
    ["modeChannel"] = "Y",
    ["channel"] = 100,
    ["system"] = 5,
    --["unitId"] = 1,
    ["bearing"] = true,
    --["frequency"] = 1061000000,
  } 
}
```
(The lines that are commented are not commented in the file but are not needed for correct function)
Using the standard DCS API setCommand function with this argument worked as expected. The main difference is that frequency is no longer used and is replaced by channel and channelMode.

**Comment by Applevangelist** [comment]
@funkyfranky Hi Frank, looks there are differences in the definition of `BEACON.Type` and `BEACON.System` between us and what is tn Hoggit.
[Hoggit](https://wiki.hoggitworld.com/view/DCS_command_activateBeacon) This might be the source of non-functionality. Can you have a look pls?

**Comment by funkyfranky** [comment]
Hoggit data is outdated. You find the types and systems in a DCS file `DCS World OpenBeta\Scripts\World\Radio\BeaconSites.lua`. Meanwhile there are even a few more that are not in moose as I just checked.

The AATACAN is also a bit outdated, yes. I think, I did some changes to make it more like the current command you get from the mission file. But maybe that was only in the `develop` branch.

I usually use `BEACON:ActivateTACAN` and not `BEACON:AATACAN`
